enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,374 @@
+and users [should migrate to IDMSes](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror-convert_image-configuration) instead.
+* Using [`ImageContentPolicy` resources](https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagecontentpolicy-config-openshift-io-v1.html)
+
+
+## Proposal
+
+### Design Details
+
+There are a couple pieces of work as part of this enhancement. All changes detailed in this enhancement proposal will be
+limited to the Windows Machine Config Operator and its sub-component, Windows Instance Config Daemon (WICD).
+
+First, WMCO will set the `config_path` variable in the containerd config file.
+Next, the operator will generate registry config files based on cluster resources. 
+And lastly, WMCO will place the generated files in the containerd config directory on each Windows instance.
+
+WMCO will use the mirrors data in `ImageDigestMirrorSet` and `ImageTagMirrorSet` resources 
+to create a set of [hosts.toml files](https://github.com/containerd/containerd/blob/main/docs/hosts.md#hoststoml-content-description---detail),
+one for each registry mirror, that will be transferred to the Windows instance. WMCO will handle the file content 
+creation and will copy over the files to each Windows node.
+
+Since WMCO is an day 2 operator (optional, not part of the OCP payload), it will pick up the mirror configuration during
+runtime regardless of whether they were set during cluster install time or at some point during the cluster's lifetime.
+
+#### Setting `config_path`
+
+WMCO will need to set the `config_path` variable in the containerd config file to tell it where to look for registry
+config settings; it is currently empty in value. We should likely set it to `C:/k/containerd/registries` -- this is 
+within our managed directories and ensures the directory contents are cleaned up during node deconfiguration.
+
+#### Config File Management
+
+WMCO will generate the required registry config files based on the IDMS/ITMS resources in the cluster.
+For new nodes, these files will be copied over to the instances on configuration, similarly to how other config files
+(kublet conf, bootstrap kubeconfig, files from ignition, etc) are treated today.
+
+For existing nodes, the operator will react to updates to the cluster's mirror settings through a new controller
+which will watch for events on IDMS/ITMS resources. This "registry controller" will generate and copy over the new
+registry config files to each Windows instance when an event is detected.
+We will fully replace the contents of the registry config directory on any change; this way, out of date registry config
+files will be removed without any extra mechanism to track files we have created in the past.
```

Comment:
> on any change

On any change to the IDMS/ITMS cluster objects.

This distinction is important as if a local change to the directory is made, WMCO has no way of detecting this, and the change will remain until a new IDMS/ITMS change is made in the cluster.

Please cover this in the risks/drawbacks section.

Reply:
> This distinction is important as if a local change to the directory is made, WMCO has no way of detecting this, and the change will remain until a new IDMS/ITMS change is made in the cluster.

Added this to the Risks section